Tools/freeze/freeze.py

```diff
@@ -217,7 +217,12 @@ def main():
 
     # locations derived from options
     version = '%d.%d' % sys.version_info[:2]
-    flagged_version = version + sys.abiflags
+    if hasattr(sys, 'abiflags'):
+        # on Windows this will make freeze traceback
```

Comment:
Sorry but I don't think that the comment is really helpful in any way. Please just remove it ;-)